Makefile

```diff
@@ -115,8 +115,32 @@ swagger: swagger/docs.go
 
 .PHONY: update-licenses
 update-licenses:
-	go get github.com/google/go-licenses
-	$(shell echo "Module | License URL | License" > THIRD-PARTY-LICENSES.md; echo "---|---|---" >> THIRD-PARTY-LICENSES.md; for i in $$(cat go.sum  | awk '{print $$1}'); do l=$$(go-licenses csv $$i 2>/dev/null); if [ $$? -ne 0 ]; then continue; fi; echo $$l | tr \, \| | tr ' ' '\n'; done | sort -u >> THIRD-PARTY-LICENSES.md)
+	go install github.com/google/go-licenses@latest
```

Comment:
This does not compile with Go 1.15 ....anyway our zot does not compile with 1.15 anyway ("make" fails).
We probably need to update in README.md file that we support Go 1.16+ (now it is Go 1.15+)

Reply:
This is being done in a different README cleanup PR #388